### PR TITLE
 Comment out some obsolete extern declarations

### PR DESCRIPTION
**Pull request by markuspf**
Fix some more of the issues noted in #935

### CODE REVIEW

**Comment by fingolfin** [comment]
Travis produced a compile error...

File: src/read.c [hunk 1]
```diff
@@ -189,7 +189,7 @@ UInt GlobalComesFromEnclosingForLoop (UInt var)
 **        |  <Var> '.' <Ident>
 **        |  <Var> '(' [ <Expr> { ',' <Expr> } ] [':' [ <options> ]] ')'
 */
-extern Obj ExprGVars;
```
Comment (fingolfin) [inline src/read.c]: This change is incorrect, `ExprGVars` is not (yet) in the global state.

**Comment by markuspf** [comment]
Yes, sorry, I was aware of that, but the weekend and a proposal got in between. Will fix shortly.

**Comment by markuspf** [comment]
There you go.

File: src/gap.c [hunk 2]
```diff
@@ -1203,10 +1203,9 @@ Obj FuncWindowCmd (
...
 */
 
-/*
-TL: Obj  ErrorLVars0;
-TL: Obj  ErrorLVars;
-TL: Int  ErrorLLevel;
+/* TL: Obj  ErrorLVars0;    
+   TL: Obj  ErrorLVars;
+   TL: Int  ErrorLLevel;
 */
```
Comment (fingolfin) [inline src/gap.c]: Is this part of the change really necessary? Seems to serve no purpose, but it'll break my PR #934 (which I'd then have to rebase once again). So, if you don't mind terribly, perhaps you could drop this one change? I'd then be happy to merge the rest as it is :-).
Thanks!

Comment (markuspf) [inline src/gap.c]: It wasn't, reverted now.

**Pull request merged**